jekyll/_cci2/orbs-faq.md

```diff
@@ -9,164 +9,66 @@ order: 20
 
 This document describes various questions and technical issues that you may find helpful when working with orbs.
 
-* TOC
-{:toc}
+## Private orbs
 
-## Downloading, Integrating, and Testing Orbs
-{:.no_toc}
+* **Question:** Can orbs be made private?
 
-* Question: How do I download, integrate and test orbs?
+* **Answer:** Currently, all orbs published to the [Orb Registry]() are open source. Please subscribe to this feature request for updates: [Feature Request: private orbs](https://ideas.circleci.com/ideas/CCI-I-606)
 
-* Answer: You can import orbs using the `orbs` stanza in version 2.1 or higher of a CircleCI configuration. For example, if you want to publish an orb called  `hello-build` in the namespace `circleci` and have a published version `0.0.1`, import an orb like the example shown below:
+## Difference between commands and jobs
 
-```
-orbs:
-     hello: circleci/hello-build@0.0.1
-```
+* **Question:** What is the difference between commands and jobs?
 
-You may then invoke elements of the orb in your configuration by referencing elements under the key `hello`. For example, if an orb has a job called `hello-build`, you can invoke that job in a workflow like the example shown below.
+* **Answer:** Both [commands]({{site.baseurl}}/2.0/reusing-config/#the-commands-key) and [jobs]({{site.baseurl}}/2.0/reusing-config/#authoring-parameterized-jobs) are elements that can be used within orbs. _Commands_ contain one or many [steps]({{site.baseurl}}/2.0/configuration-reference/#steps), which contain the logic of the orb. Commands generally execute some shell code (bash). _Jobs_ are a definition of what steps/commands to run and within what [executor]({{site.baseurl}}/2.0/reusing-config/#executors) to run them. _Commands_ are invoked at the job level. _Jobs_ are invoked at the _[Workflow]({{site.baseurl}}/2.0/workflows/#workflows-configuration-examples)_ level.
 
-```
-workflows:
-  info:
-    jobs:
-      - hello/hello-build
-```
+## Using Orbs on CircleCI Server
 
-CircleCI publishes a web-based registry viewer so orbs documentation can be auto-generated. You can always pull the source of orbs directly as well. For example, you can run `circleci orb source circleci/hello-build@0.01`
+* **Question:** Can orbs be used on a private installation of CircleCI server?
 
-## Build Error When Testing Locally
```

Comment:
@KyleTryon just wondering about the reasons for removing most of the existing FAQs? If they are not obsolete in some way I would like to add them back in, in addition to the ones you've added here, as we've had good feedback on this doc in the past, suggesting some people have found them useful.

Reply:
For this one here, we do not want to encourage testing locally in this way at the time. We are producing new testing documentation what will not make mention of running jobs locally. We will instead focus on testing code locally (which was previously not possible) and testing the orb itself directly on CircleCI (rather than a local circleci job). This also makes it easier for the CPE team to support this style of testing as we have less control over the local circleci build command.

Reply:
Ah yep that makes sense!

Reply:
My comment RE FAQs was regarding the entire doc, so the rest of the deleted FAQs too.

Reply:
@KyleTryon ^^